client/src/plus/offer-overview/offer-overview-subscribe/index.tsx

```diff
@@ -252,6 +252,13 @@ function OfferOverviewSubscribe() {
           <OfferDetails offerDetails={PLUS_10} period={period}></OfferDetails>
         </div>
       </section>
+      <p className="plus-for-companies">
+        * Do you need MDN Plus for your company?{" "}
+        <a href="https://docs.google.com/forms/d/15YimonAiA9ca-JrGfxgRstYEuQsVUuyzGH1_0RbpSPU/edit">
```

Comment:
Are we sure this is the correct public link with the `/edit` suffix?

If I click on it, I see a "Request edit access" button on the bottom right:
<img width="1052" alt="image" src="https://user-images.githubusercontent.com/495429/160559075-bad73180-0aab-47e1-ae43-c02f47a5d59c.png">

(But maybe this is because I'm using my Mozilla account.)

Reply:
FWIW https://docs.google.com/forms/d/e/1FAIpQLScnWZdBBxr8ilhOknvunIWHHWpoAOqoEvIzhXBB3Py8SbDTSw/viewform (with the `/viewform` suffix) doesn't show the "Request edit access" button.